exploration/function-composition-part-1.md

```diff
@@ -494,7 +497,10 @@ of the runtime behavior of function composition.
 
 then the formatted result is "0.33 0.33333".
 
-**Model 2:**
+**Simple Model (Model 2):**
+
+We call this the "simple" model because the result of the function
+is simpler than in the preservation model.
```

Comment:
This is unclear? What is simple or simpler?

```suggestion
### Formatted Value Model

This model preserves the formatted value of the function,
but not the options that were passed to the function.
```

Reply:
Sure, that's a good name. I applied this change and updated the rest of the document to reflect that.